File: 프로그래머스/0/120809. 배열 두배 만들기/배열 두배 만들기.py
def solution(numbers):
    #파이썬에서는 배열크기를 초기화 할 때 [0]*len(numbers)를 쓴다.
    answer = [0]*len(numbers)
    
    for i in range (len(numbers)):
     answer[i] = numbers[i] * 2
    
    return answer

File: 프로그래머스/0/120809. 배열 두배 만들기/배열 두배 만들기.js
function solution(numbers) {
    //let [] answer = new let [numbers.length]; ㅋㅋㅋㅋ이건 자바잖아요.. 변수만 let으로 해주면 끝이냐고
    let answer = new Array(numbers.legnth);
    for(let i=0; i<numbers.length; i++){
        answer[i] = numbers[i]*2;
    }
    
    return answer;
}